content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,265 @@
+   communicating stability, through a metadata file, that can be discovered and
+   parsed in a programmatic way. The exact format will be defined through an
+   OTEP and incorporated into the specification. As part of this, we'll be
+   normalizing stability levels across components, including semantic
+   conventions by introducing alpha/beta stability to that effort.
+2. Stability requirements will be expanded to include more requirements around
+   documentation and where it's hosted, example code, performance benchmarks
+   (where applicable), implementation cookbooks, and other artifacts as
+   necessary.
+3. Stable components must only include other stable components by default. A new
+   global configuration option shall be introduced that allows adopters to
+   choose a desired minimum stability level, with a default value of
+   `[stable, rc, beta]`.
+
+This is a big change for maintainers, especially those who have shipped v1+ of
+their libraries. We would deeply appreciate your feedback on this proposal in
+the [discussion](https://github.com/open-telemetry/community/discussions/3098).
+
+## 2. Semantic Convention Stability Normalization
+
+As mentioned above, we would like to adjust the semantic convention stability
+process in order to provide greater consistency across projects, but also to
+better communicate the interplay between telemetry stability and implementation
+stability.
+
+### Background
+
+Semantic conventions are a slow, deliberative, and ultimately user-driven
+project. Maintainers must grapple with the reality of end-user telemetry systems
+and their varying architectures. While OpenTelemetry's design is situated around
+a continuous stream of highly intercorrelated signals, users may choose a
+variety of options for consuming that data. The decision of a storage and
+analysis engine for telemetry has a variety of interplay with semantic
+conventions, and maintainers must consider the tradeoffs between cardinality,
+dimensionality, and specificity of attributes when designing conventions and
+stabilizing them.
+
+The downside of this is that progress on semantic conventions can be slow, and
+this slowness impacts all consumers of the conventions. Many instrumentation
+libraries are currently stuck on pre-release versions because they depend on
+experimental semantic conventions. Outside contributors are stuck between
+emitting unspecified telemetry or trying to engage in the process, which
+requires a long commitment. Finally, we're internally inconsistent in
+instrumentation across the project; Some libraries are mapped to conventions,
+others exist independently of it.
+
+### What's Changing?
+
+Our goals here are designed to achieve two outcomes.
+
+1. Semantic conventions should have more nuanced stability levels in order to
+   unblock upstream development and gain real world feedback from users.
+2. Semantic conventions should be more federated; We don't want the project to
```

Comment:
"We" is confusing again. Is it the project maintainers or GC or TC or all of the above?